haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku, \
+intended to be a full-featured organizer. "
```

Comment:
Oops, you forgot to remove the space near the end of line 3.

Hmm, the comma between `Eventual is a personal time management system for Haiku` and `intended to be a full-featured organizer.` does not sound very nice. What do you think of these possible variants?
```
DESCRIPTION="Eventual is a personal time management system for Haiku and is \
intended to be a full-featured organizer."
```
```
DESCRIPTION="Eventual is a personal time management system intended to be a \
full-featured organizer."
```

(BTW, thanks for the previous update in `BUILD()` :)

Reply:
Perhaps the first one? We shouldn't forget Haiku!!!